### PR TITLE
update Debezium script for v2 python

### PR DESCRIPTION
**Pull request by hythloda**

### CODE REVIEW

File: debezium/scripts/demo.py [hunk 1]
```diff
@@ -36,133 +38,133 @@ def make_cdc_table(table_name:str):
...
+        ts_col='received_at',
+        window=10*minute_in_nanos,
+        bool_col='in_last_10min'
     ).where(
-        'in_last_10min = true'
-    ).updateView(
-        'received_at_minute = lowerBin(received_at, minute_in_nanos)'
+        ['in_last_10min = true']
+    ).update_view(
+        ['received_at_minute = (DateTime)lower_bin(received_at, minute_in_nanos)']
```
Comment (jcferretti) [inline debezium/scripts/demo.py:102]: I am confused as for why we would need a cast here.  Can we figure out what the types are?

File: debezium/scripts/demo.py [hunk 2]
```diff
@@ -36,133 +38,133 @@ def make_cdc_table(table_name:str):
...
 item_summary = items \
-    .view('item_id = id', 'name', 'category') \
-    .naturalJoin(purchases_by_item, 'item_id') \
-    .naturalJoin(pageviews_by_item, 'item_id') \
-    .dropColumns('item_id') \
-    .moveColumnsDown('revenue', 'pageviews') \
-    .updateView('conversion_rate = orders / (double) pageviews')
+    .view(['item_id = id', 'name', 'category']) \
+    .natural_join(purchases_by_item, on = ['item_id']) \
+    .natural_join(pageviews_by_item, on = ['item_id']) \
```
[the remark this answers is not in the document]
Comment (hythloda) [reply debezium/scripts/demo.py:75]: I am missing seeing a repeat.  We join both purchases_by_item and pageviews_by_item

File: debezium/scripts/demo.py (hunk 1 above)
Comment (hythloda) [reply debezium/scripts/demo.py]: resolved that parser would be useful for situations like this. In the mean time using lowerBin.

File: debezium/scripts/demo.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (jcferretti) [reply debezium/scripts/demo.py:75]: Sorry my bad.  Need new eyes.

File: debezium/scripts/demo.py [hunk 3]
```diff
@@ -36,133 +37,133 @@ def make_cdc_table(table_name:str):
...
+    ]).agg_by([
+            agg.sum_(['lifetime_value = purchase_total']),
+            agg.count_('purchases')
+        ],
         'user_id'
-    ) \
-    .where('lifetime_value > 10000') \
-    .naturalJoin(users, 'user_id = id', 'email') \
-    .view('id = user_id', 'email', 'lifetime_value', 'purchases')  # column rename and reorder
+    )\
```
Comment (jcferretti) [inline debezium/scripts/demo.py:148]: space between close paren and backslash

File: debezium/scripts/demo.py [hunk 4]
```diff
@@ -36,133 +37,133 @@ def make_cdc_table(table_name:str):
...
-    .updateView(
+    .update_view([
         'purchase_total = purchase_price * quantity'
-    ).aggBy(
-        as_list([
-            agg.AggSum('lifetime_value = purchase_total'),
-            agg.AggCount('purchases'),
-        ]),
+    ]).agg_by([
+            agg.sum_(['lifetime_value = purchase_total']),
```
Comment (jcferretti) [inline debezium/scripts/demo.py:144]: why the trailing underscores here?

Comment (jcferretti) [reply debezium/scripts/demo.py:144]: We discussed in slack.

File: debezium/scripts/demo.py (hunk 3 above)
Comment (hythloda) [inline debezium/scripts/demo.py:148]:
```suggestion
    ) \
```

**Review by jcferretti — APPROVED** [review]

**Pull request merged**